sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -139,22 +139,26 @@ def list_configuration_settings(
         *,
         key_filter: Optional[str] = None,
         label_filter: Optional[str] = None,
+        tags_filter: Optional[List[str]] = None,
         accept_datetime: Optional[Union[datetime, str]] = None,
         fields: Optional[List[str]] = None,
         **kwargs: Any,
     ) -> ItemPaged[ConfigurationSetting]:
         """List the configuration settings stored in the configuration service, optionally filtered by
         key, label and accept_datetime.
 
-        :keyword key_filter: filter results based on their keys. '*' can be
+        :keyword key_filter: Filter results based on their keys. '*' can be
             used as wildcard in the beginning or end of the filter
         :paramtype key_filter: str or None
-        :keyword label_filter: filter results based on their label. '*' can be
+        :keyword label_filter: Filter results based on their label. '*' can be
             used as wildcard in the beginning or end of the filter
         :paramtype label_filter: str or None
-        :keyword accept_datetime: retrieve ConfigurationSetting existed at this datetime
+        :keyword tags_filter: A filter used to query by tags. Default value is None.
+        :paramtype tags_filter: list[str] or None
+        :keyword accept_datetime: Retrieve ConfigurationSetting existed at this datetime
         :paramtype accept_datetime: ~datetime.datetime or str or None
-        :keyword list[str] fields: specify which fields to include in the results. Leave None to include all fields
+        :keyword fields: Specify which fields to include in the results. Leave None to include all fields
```

Comment:
You don't "leave None" to include all fields - you don't specify the value. So if is better phrased as something like "If specified <something something>. If not specified, includes all field."